src/ida/ida_impl.h

```diff
@@ -449,6 +450,11 @@ sunrealtype IDAWrmsNorm(IDAMem IDA_mem, N_Vector x, N_Vector w,
 
 int idaNlsInit(IDAMem IDA_mem);
 
+/* Utility function to tell IDA to free the user data.
+   This is used by the Python interfaces. */
+
+int IDASetOwnUserData(void* ida_mem, sunbooleantype own_user_data);
```

Comment:
IDA is inconsistent about this but to follow the general convention prefix with `ida`
```suggestion
int idaSetOwnUserData(void* ida_mem, sunbooleantype own_user_data);
```